src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/MålgruppeValidering.kt

```diff
@@ -27,4 +34,96 @@ object MålgruppeValidering {
             "målgruppe=$målgruppeType er ikke gyldig for $stønadstype"
         }
     }
+
+    fun vurderAldersvilkår(
+        vilkårperiode: LagreVilkårperiode,
+        stønadstype: Stønadstype,
+        grunnlagsdata: Grunnlagsdata,
+    ): VurderingAldersVilkår {
+        val fødselsdato = grunnlagsdata.grunnlag.fødsel?.fødselsdato
+
+        feilHvis(fødselsdato == null) { "Kan ikke vurdere aldersvilkår uten å vite fødselsdato til bruker" }
+
+        val gyldig: SvarJaNei? =
+            when (stønadstype) {
+                Stønadstype.BARNETILSYN, Stønadstype.LÆREMIDLER ->
```

Comment:
Trenger vi å sjekke stønadstype her?

Reply:
Eller er det fordi vi skal sjekke aldersvilkår for dagpenger på noen stønadstyper? 
Burde man snu på denne switchen?

Reply:
Fjernet switch på Stønadstype: https://github.com/navikt/tilleggsstonader-sak/pull/586/commits/cda2738da36a4a27baedfdc10366266672f0560c